### PR TITLE
Fix flutter qr code

### PR DESCRIPTION
**Pull request by maxammann**

### CODE REVIEW

File: frontend/lib/identification/card_detail_view/verification_code_view.dart [hunk 1]
```diff
@@ -52,6 +52,12 @@ class _VerificationCodeViewState extends State<VerificationCodeView> {
         final padding = min(constraints.maxWidth, constraints.maxHeight) < 400 ? 12.0 : 24.0;
         return Consumer<ActivationCodeModel>(
           builder: (context, cardDetailsModel, child) {
+            final qrCode = qr.QrCode.fromUint8List(
+                          data: const QrCodeUtils().createDynamicVerificationQrCodeData(activationCode, otpCode.code),
+                          errorCorrectLevel: qr.QrErrorCorrectLevel.L,
+                        );
+            qrCode.make();
```
Comment (michael-markl) [inline frontend/lib/identification/card_detail_view/verification_code_view.dart:59]: weird api...

**Comment by f1sh1918** [comment]
can you just describe the issue that you fix. I dunno what to test here

**Comment by maxammann** [comment]
Open the Ausweisen tab with an active card. If it works everything is fine. No need to test through I think.

**Review by f1sh1918 — APPROVED** [review]
tested on android real device

**Pull request merged**